recipes/napari-tiler/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set name = "napari-tiler" %}
+{% set version = "0.0.9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/napari-tiler-{{ version }}.tar.gz
+  sha256: d13096502205fb4c5e9ca54656b350ec3e2b7c104162910d45f9bb6f0e09a78d
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - poetry-core
+    - python
+  run:
+    - importlib-metadata <4.3  # [py<38]
+    - napari-plugin-engine >=0.2.0,<0.3.0
+    - napari-tools-menu >=0.1.7,<0.2.0
+    - numpy >=1.21.4,<2.0.0
```

Are these autogenerated from the `~=` operator? I think it's safe to leave the upper bound open, but I'd like to hear from @tdmorello. As in:

```suggestion
    - napari-plugin-engine >=0.2.0
    - napari-tools-menu >=0.1.7
    - numpy >=1.21.4
```

---

Yeah I think so, but we could probably leave the bound open :)

---

It should be fine to leave it open like you suggest. I don't specify upper bounds in the pyproject.toml.